networking/networking.go

```diff
@@ -98,24 +104,28 @@ func Setup(rktRoot string, contID types.UUID) (*Networking, error) {
 
 	err = withNetNS(n.contNS, n.hostNS, func() error {
 		n.nets, err = n.setupNets(n.contNSPath, nets)
-		return err
+		if err != nil {
+			return err
+		}
+
+		if len(n.nets) == 0 {
+			return fmt.Errorf("no nets successfully setup")
```

Comment:
hrm, this error message doesn't really help much. I understand what is happening but something like "no networks configured" or something might be better.